.travis.yml

```diff
@@ -11,22 +11,30 @@ matrix:
   allow_failures:
     - env: SHELL=zsh
 script:
-  - >
-    if ${LINT:-false}; then
-      shellcheck -s sh bin/shpec  # Just use th `sh` dialect
-    else
-      time $SHELL bin/shpec shpec/shpec_shpec.sh
-    fi
+  - |
+    ${LINT:-false} &&
+      exec shellcheck -s sh bin/shpec  # `sh` refers to POSIX
+    $SHELL bin/shpec shpec/shpec_shpec.sh
```

Comment:
I believe that the subsequent tests aren't run because the `exec` is terminating the shell after that command. In doing so, we're not letting travis end the session (nor give any output). I would rather have the more explicit if/else, and let Travis end the session itself.

Also @hlangeveld for what it's worth, specifying `-f json` will at least give an empty array upon success. I say we should either do that, or something like 
```bash
shellcheck -s sh bin/shpec && echo "shellcheck success"
```